libraries/Microsoft.Bot.Builder.Ai.Translation/Microsoft.Bot.Builder.Ai.Translation.csproj

```diff
@@ -18,7 +18,6 @@
 	  <PackageId>Microsoft.Bot.Builder.Ai.Translation</PackageId>
     <Description>Classes for using Translation Cognitive Services on the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using Microsoft Cognitive services.</Summary>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Yes we want to remove from here, but we really need to _move_ to the conditional property group for `Release`.